internal/cli/cmd/cluster/describe.go

```diff
@@ -225,44 +225,42 @@ func showEndpoints(c *appsv1alpha1.Cluster, svcList *corev1.ServiceList, out io.
 	tbl.Print()
 }
 
-func showDataProtection(backupPolicies []dpv1alpha1.BackupPolicy, backups []dpv1alpha1.Backup, out io.Writer) {
-	if len(backupPolicies) == 0 {
+func showDataProtection(backupPolicies []dpv1alpha1.BackupPolicy, backupSchedules []dpv1alpha1.BackupSchedule, out io.Writer) {
+	if len(backupPolicies) == 0 || len(backupSchedules) == 0 {
 		return
 	}
-	tbl := newTbl(out, "\nData Protection:", "AUTO-BACKUP", "BACKUP-SCHEDULE", "TYPE", "BACKUP-TTL", "LAST-SCHEDULE", "RECOVERABLE-TIME")
+	tbl := newTbl(out, "\nData Protection:", "BACKUP-REPO", "AUTO-BACKUP", "BACKUP-SCHEDULE", "BACKUP-METHOD", "BACKUP-RETENTION")
 	for _, policy := range backupPolicies {
 		if policy.Annotations[dptypes.DefaultBackupPolicyAnnotationKey] != "true" {
 			continue
 		}
 		if policy.Status.Phase != dpv1alpha1.AvailablePhase {
 			continue
 		}
-		//	ttlString := printer.NoneString
-		//	backupSchedule := printer.NoneString
-		//	backupType := printer.NoneString
-		//	scheduleEnable := "Disabled"
-		//	if policy.Spec.SchedulePolicy.Snapshot != nil {
-		//		if policy.Spec.SchedulePolicy.Snapshot.Enable {
-		//			scheduleEnable = "Enabled"
-		//			backupSchedule = policy.Spec.SchedulePolicy.Snapshot.CronExpression
-		//			backupType = string(dpv1alpha1.BackupTypeSnapshot)
-		//		}
-		//	}
-		//	if policy.Spec.SchedulePolicy.Datafile != nil {
-		//		if policy.Spec.SchedulePolicy.Datafile.Enable {
-		//			scheduleEnable = "Enabled"
-		//			backupSchedule = policy.Spec.SchedulePolicy.Datafile.CronExpression
-		//			backupType = string(dpv1alpha1.BackupTypeDataFile)
-		//		}
-		//	}
-		//	if policy.Spec.Retention != nil && policy.Spec.Retention.TTL != nil {
-		//		ttlString = *policy.Spec.Retention.TTL
-		//	}
-		//	lastScheduleTime := printer.NoneString
-		//	if policy.Status.LastScheduleTime != nil {
-		//		lastScheduleTime = util.TimeFormat(policy.Status.LastScheduleTime)
-		//	}
-		//	tbl.AddRow(scheduleEnable, backupSchedule, backupType, ttlString, lastScheduleTime, getBackupRecoverableTime(backups))
+		backupRepo := printer.NoneString
+		backupMethod := printer.NoneString
+		backupSchedule := printer.NoneString
+		backupRetention := printer.NoneString
+		scheduleEnable := "Disabled"
+		for _, schedule := range backupSchedules {
+			if schedule.Spec.BackupPolicyName == policy.Name {
```

Comment:
Reduce the nesting depth, if name is not equal, return fast. And `if shcedulePolicy.Enabled != nil && *shcedulePolicy.Enabled` is similar.